report/PreProject/PreProject.tex

```diff
@@ -0,0 +1,45 @@
+The method is called \textbf{Essence}, and the main focus is the pre-project, e.g. before we start a project.
+Essence is not a full work framework, such as Scrum or RUP, but is used together with these.
+So you will typically use Essence before, to get a better start and understanding of the problem, and then go over to Scrum, where you will have Sprints. 
+Essence then has a \textbf{configuration table}, which we will highlight later in the report, but the idea is, when a sprint is done, then you will review this table, and if there are major changes to it, then we will need to change focus.
+This is where the Agile process helps, as it should be easy to change the focus, whereas it can be harder in Plan-driven process, however, in our opinion you could use Essence in these processes as well. 
+We will try to follow the principles as best as we can, however, this is our interpretation of it, so there might be some misunderstandings.
+In the following sections, will we go over the Structure of Essence.
+
+So why do we want to use it?
+The main reason why, is that it seems to be a good tool to use, to get a better start at a project, while we still can utilize an Agile process, such as Scrum and XP.
+Furthermore, it gives a good insight into are the project on the right track and are we developing the right solution.
+So the overview can be easier to get, and you do do not have to piece it all together, as all the information you need, is in the configuration table.
+
+\subsection{Structure in Essence}
+In Essence, we have \textbf{Pre-project}, which is the main focus of Essence, in our opinion.
+The pre-project is as the name state before a project starts. 
+The overall goal is to identify some key technologies and other important information we might need to solve the problem at hand.
+The way we do this is that we have four steps we need to get through.
+These are:
+
+\begin{itemize}
+    \item Find a \textbf{Challenge}
+    \item Find the \textbf{Ecology Interfaces}
+    \item Find the \textbf{Leverage Points}
+    \item Find the \textbf{Initial Problem}, from the Challenge, Ecology Interfaces and Leverage Points
+\end{itemize}
+
+First, we need to find a challenge.
+A challenge is a problem we want to solve, however, we do not know how we might solve it, what technologies we need to use, what platforms we should use and so forth.
+You can think of it as an Initial problem statement.
+It is narrowed down a bit, however, not enough to be a problem, that we should invest a lot of time and resources in.
+
+After we have identified a challenge, we need to find the Ecology Interfaces and Leverage Points.
+These two processes are similar, however, the focus is on different aspects.
+Ecology focuses more on the \textbf{external} aspects of the project, meaning that if we can use a piece of software another company has made, and they have a trivial interface, then it will make our workload lower, than if we needed to make it ourselves.
```

Comment:
```suggestion
Ecology focuses more on the \textbf{external} aspects of the project, meaning that if we can use a piece of software another company has made, and they have a trivial interface, then it will make our workload smaller, than if we needed to make it ourselves.
```